profiler/src/main/java/com/splunk/opentelemetry/profiler/JfrActivator.java

```diff
@@ -151,13 +153,19 @@ private void activateJfrAndRunForever(Config config) {
     dirCleanup.registerShutdownHook();
   }
 
-  private LogsExporter buildExporter() {
-
+  private LogsExporter buildExporter(Config config) {
     ResourceLogsAdapter adapter = buildResourceLogsAdapter();
-    return OtlpLogsExporter.builder()
-        .setAdapter(adapter)
-        .addHeader("Extra-Content-Type", "otel-profiling-stacktraces")
-        .build();
+
+    OtlpLogsExporterBuilder builder =
+        OtlpLogsExporter.builder()
+            .setAdapter(adapter)
+            .addHeader("Extra-Content-Type", "otel-profiling-stacktraces");
+
+    String ingestUrl = config.getProperty(CONFIG_KEY_INGEST_URL);
+    if (ingestUrl != null) {
+      builder.setEndpoint(ingestUrl);
+    }
+    return builder.build();
```

Comment:
do we have any way to write a unit test for this, at least to make sure it doesn't blow up?

Reply:
I'm not really sure of a good approach. We could put a setter on the builder and ensure that in the non-null case it got set, but I'm not sure how much that helps us....do you?

Reply:
maybe make this method test-accessible and put in null & not-null into the config and make sure you get an non-null activator out the other side?

Reply:
or separate exporter building into its own testable class?

Reply:
ok, took a stab at it.  Lemme know what you think.